database/diesel.toml

```diff
@@ -0,0 +1,9 @@
+# For documentation on how to configure this file,
+# see https://diesel.rs/guides/configuring-diesel-cli
+
+[print_schema]
+file = "src/postgres/schema.rs"
+custom_type_derives = ["diesel::query_builder::QueryId"]
```

Comment:
Could you clarify what does it mean?

Reply:
It is generated automatically by Diesel CLI.
https://diesel.rs/guides/configuring-diesel-cli